packages/itwinui-css/src/toast/toast.scss

```diff
@@ -52,13 +52,13 @@
   z-index: 99999;
 
   @media screen and (max-width: 400px) {
```

Comment:
Should this check for `max-width` be changed to `max-inline-size`?

Reply:
done

Reply:
no, logical properties do not work inside media queries.

Reply:
reverted